.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
>Let me use my larger-than-standard-VGA-resolution screen properly.

I think the 80 (or 90) character limitation goes beyond screen space capacity. It enhances the readability and *ensures* that the lines won't wrap on any screen (maybe ones that are not larger-than-standard).

>I would be really, really unhappy if I was asked to have 8.5/10 on pylint in order to be able to contribute to mbed-os. A lot of pylint rules are pseudo-arbitrary at best. 

 Many style rules are "pseudo-arbitrary" (vestiges of some old tech, or the inventor, or whatever), but I think the standardization that adhering to these rules provides is the key.

If the rules are clearly defined, and the rule violations are obvious, I don't see why it would make contributors upset.